CHANGELOG.md

```diff
@@ -1,3 +1,93 @@
+v1.20.0
+===
+
+This is a comparatively small minor release that adds support for Kafka 4.1,
+adds three new APIs, fixes four bugs (read below to gauge importance), has a
+few improvements, and **switches the client from a default of 0ms linger to a
+default of 10ms linger**.
+
+Also of note: a new `srfake` package has been created so you can run a fake
+Schema Registry server in your CI tests (thank you [@weeco](https://github.com/weeco)).
+This complements the existing `kfake` package that allows you to run a fake
+in-memory Kafka "cluster" for unit testing. If you did not know of either of these,
+check them out! `kfake` supports many Kafka features, but transactions are still WIP.
+All franz-go tests except transaction based tests pass against a kfake "cluster",
+so odds are, it'll work for you.
+
+## Behavior changes
+
+* This library now lingers by default for 10ms. You can switch back to 0ms
+  lingering by adding `kgo.ProducerLinger(10*time.Millisecond)` to your options
```

Comment:
@hhromic yes this should be 0*time.Millisecond (or just 0). Thanks for catching, not sure where your comment is though since I'm not seeing it here.